boards/common/cc2538/include/cfg_rtt_default.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_cc2538
+ * @{
+ *
+ * @file
+ * @brief       Default RTT configuration for cc2538 boards
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ */
+
+#ifndef CFG_RTT_DEFAULT_H
+#define CFG_RTT_DEFAULT_H
+
+#include <stdint.h>
+
+#include "cpu.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name RTT configuration
+ * @{
+ */
+#define RTT_DEV             SMWDTHROSC
+#define RTT_IRQ             SM_TIMER_ALT_IRQn
+#define RTT_IRQ_PRIO        1
+#define RTT_ISR             isr_sleepmode
+#define RTT_MAX_VALUE       (0xffffffff)
```

Comment:
I think this might as well be moved into `periph_cpu.h`

Reply:
True, I only really need:

```
#if SYS_CTRL_OSC32K_USE_XTAL
/* Frequency of XOSC off by default */
#define RTT_FREQUENCY       (32768U)    /* in Hz. For changes see `rtt.c` */
#else
/* Frequency of RCOSC on by default */
#define RTT_FREQUENCY       (32753U)    /* in Hz. For changes see `rtt.c` */
#endif
```
 so I could move it back, should I still keep a `cfg_rtt_default.h` or move the above lines to `cfg_timer_default`?

Reply:
is it even possible to use a non-32kHz XTAL here?
If you *must* use a 32768 Hz oscillator here, there is nothing for the board to configure other than defining `SYS_CTRL_OSC32K_USE_XTAL` if it's there - this could also go to `periph_cpu.h`.

Reply:
> s it even possible to use a non-32kHz XTAL here?
> If you must use a 32768 Hz oscillator here, there is nothing for the board to configure other than defining SYS_CTRL_OSC32K_USE_XTAL if it's there - this could also go to periph_cpu.h.

You can use the 32Khz RCOSC which is `32753KHz` which is a little less precise, but more power efficient than the XTAL

Reply:
Sure, but it's always either 32768 Hz or 32753 Hz, so
```C
#if SYS_CTRL_OSC32K_USE_XTAL
/* Frequency of XOSC off by default */
#define RTT_FREQUENCY       (32768U)    /* in Hz. For changes see `rtt.c` */
#else
/* Frequency of RCOSC on by default */
#define RTT_FREQUENCY       (32753U)    /* in Hz. For changes see `rtt.c` */
#endif
```
could live inside `periph_cpu.h` too.

Reply:
Meh, that won't work because `periph_cpu,h` is included by `periph_conf.h` before `SYS_CTRL_OSC32K_USE_XTAL` gets set.

Reply:
Yep, maybe I should have been clearer in the PR description. All the `RTT` confifugration was in
`periph_cpu.h` before, and before you would set `SYS_CTRL_OSC32K_USE_XTAL` though a `CFLAG`

But with #13673, you need to set it in `periph_conf`, so that is why at least the above code needs to be included there as well (or after), but `periph_cpu` is included before.

Reply:
So should I move part of rtt config to `periph_cpu` again?

Reply:
Yea I think you would not need a `cfg_rtt_default.h`/any board specific RTT config at all then.

Reply:
Yep, there is nothing specific needed, but I will still need to define `#define RTT_FREQUENCY       (CLOCK_OSC32K)` in the `cfg_clk_default.h` if that is Ok for you

Reply:
Oh I hadn't understood that. It works, so I'll do that!